website/hugo.toml

```diff
@@ -0,0 +1,47 @@
+baseURL = 'https://llvm.github.io/wg-hlsl'
+languageCode = 'en-us'
+title = 'wg-hlsl'
+theme = 'hugo-theme-techdoc'
+
+[params]
+  menu_style = "slide-menu"
+  
+[markup]
+  [markup.goldmark]
+    [markup.goldmark.renderer]
+      unsafe = true
+
+ignoreLogs = ['warning-goldmark-raw-html']
```

Comment:
This sounds scary to me, but it allows us to use HTML tags in our documents without generating warnings, which can be reasonable escape hatch when markdown isn't sufficient.

Reply:
For line breaks, HTML is not required. Other ways to line break in GitHub-flavored Markdown are:
- Ending a line with two spaces
- Ending a line with a backslash

Break this\
line!

Or break this  
line!